ansible/roles/containers/templates/atfncat.def

```diff
@@ -0,0 +1,118 @@
+Bootstrap: localimage
+From:  {{ base_container_image }}
+
+%labels
+  Maintainer Tinus Cloete
+
+%help
+  atfncat container
+  Test Command: psrcat -db_file /software/psrcat/psrcat.db -c "name p0 dm" 0437-4715
+
+%environment
+  export PATH=/software/psrcat:${PATH}
+  export PSRCAT_FILE=/software/psrcat/psrcat.db
+
+%files
+  # pass
+
+%post
+  # Create Installation Directories and export paths. This is needed as part of post.
+  # %environment scriptlet does not define these paths during %post, only after.
+  mkdir -p /software
+  export SOFTWARE_PATH=/software
+  export DEBIAN_FRONTEND=noninteractive
+
+  # Update apt
+  apt-get update
+  apt-get update -y && apt-get dist-upgrade -y
+
+
+  # Install dependencies
+  apt-add-repository universe
+
+  apt-get install software-properties-common -y
```

Comment:
Might be redundant if already in the base container? Doesn't hurt here though if it's not. Probably safer in some ways…